essentials/src/api/subxt_wrapper.rs

```diff
@@ -473,7 +471,11 @@ async fn subxt_get_block_ts(api: &OnlineClient<PolkadotConfig>, hash: H256) -> R
 }
 
 async fn subxt_get_block(api: &OnlineClient<PolkadotConfig>, maybe_hash: Option<H256>) -> Result {
-	Ok(Response::MaybeBlock(api.rpc().block(maybe_hash).await?.map(|response| response.block)))
+	let block = match maybe_hash {
+		Some(hash) => api.blocks().at(hash).await?,
+		None => api.blocks().at_latest().await?,
+	};
```

Comment:
nit:

```suggestion
	let block = api.blocks().at_or_latest(maybe_hash).await?;
```

Reply:
I'd love to use `at_or_latest` but it is a private method :-)

Reply:
Ahh I see, we removed in favor of `at` and `at_latest`